DeveloperLabs/MyEmployees/MyEmployees/Form1.cs

```diff
@@ -32,10 +36,79 @@ private void Form1_Load(object sender, EventArgs e)
         {
             LoadConfig();
             LoadData();
+            AddAppInstaller(getAppInstallerUri());
             CheckKioskMode();
             //await CheckForUpdates();
         }
 
+        // Retrieves the URI of the .appinstaller which is embedded in the application (MyEmployees) package
+        public static string getAppInstallerUri()
+        {
+            string uriPath = null;
+            try
+            {
+                uriPath = System.IO.Path.GetDirectoryName(System.Windows.Forms.Application.ExecutablePath) + "\\..\\Update.appinstaller";

```

Comment:
It would have been nice to avoid the relative path usage here. Not blocking but as a best practice relative paths can sometimes have unintended consequences if the executable location were ever to change.